### ISSUE TITLE
Feature: add ability to capture audio

### ISSUE THREAD

**Issue by silkcom** [rasha08/react-canvas-recorder#1]
Would love the ability to capture audio with this as well.  I'm not sure if that's what captureMediaStream does (so maybe this is more of a documentation request?).

**Comment by rasha08** [comment]
Of course, @silkcom, will include feature/docs in the next two or three days.

**Comment by preetparm** [comment]
Hi Bro, can we capture audio from system ? not from mike!
